### ISSUE TITLE
Remove Contract Address QR code function

### ISSUE THREAD

**Issue by tab00** [AlphaWallet/alpha-wallet-android#3306]
I found that the token address of a token can be found in the "My Wallet Address" screen by clicking on the Ethereum logo on the top right.

Making it possible to show a QR code of the contract address is a bad idea because then someone could scan and sends tokens to the contract, instead of the intended recipient. The Ethereum logo could be accidentally pressed. Such tokens can then end up being stuck in there forever as usually token contract aren't programmed with a custom function for withdrawing tokens.

Usually there's no reason to be sending tokens into a token contract.

**Comment by JamesSmartCell** [comment]
Yep ok. This is actually in here for development purposes. Now that we can access it from the token info itself this can be removed.

**Comment by JamesSmartCell** [comment]
There's currently no way to access that option, it's been deprecated, but that code should be removed too.
